media-gfx/dmtx-utils/dmtx_utils-0.7.4.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="A software for reading and writing Data Matrix barcodes"
+DESCRIPTION="This is open source software for reading and writing Data Matrix \
+barcodes on Linux, Unix, OS X, Windows, and mobile devices. At its core \
+libdmtx is a native shared library, allowing C/C++ programs to use \
+its capabilities without extra restrictions or overhead."
+HOMEPAGE="http://libdmtx.sourceforge.net/"
+COPYRIGHT="2009-2017 Mike Laughton"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/libdmtx/dmtx-utils-$portVersion.tar.gz"
+CHECKSUM_SHA256="27104dba2a53979c30a0788f4c9ccbf5e30fbe382b9bb6942928f9f869f528b6"
+SOURCE_DIR="dmtx-utils-$portVersion"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+# On x86_gcc2 we don't want to install the commands in bin/<arch>/, but in bin/.
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+commandSuffix=
+commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	dmtx_utils$secondaryArchSuffix = $portVersion
+	cmd:dmtxquery$commandSuffix = $portVersion
+	cmd:dmtxread$commandSuffix = $portVersion
+	cmd:dmtxwrite$commandSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	imagemagick$secondaryArchSuffix
+	lib:libdmtx$secondaryArchSuffix
+	"
+BUILD_REQUIRES="
```

Comment:
Oops, you removed the empty line between REQUIRES and BUILD_REQUIRES... but we like to have that one ;-)